src/main/resources/stubs/net/net.gobra

```diff
@@ -294,12 +294,14 @@ type DNSError struct {
 pred (e *DNSError) Mem() { acc(e) }
 
 preserves e != nil ==> acc(e, 1/1000)
+trusted
```

Comment:
Does this allow you to verify this file with Gobra as the body is ignored and thus the exception does not occur? If so, please add a comment next to the trusted annotation as I don't think I would figure out what its purpose is when I'm not seeing that you've also added the comment below

Reply:
I actually found the root cause of this issue and fixed this ^^ so, it is not trusted anymore, and I even manage to make the code simpler to fix this.